internal/cmd/internal/olmv1/catalog/alpha_inspect.go

```diff
@@ -0,0 +1,72 @@
+package catalog
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/operator-framework/kubectl-operator/internal/pkg/catalogd/fetcher"
+	"github.com/operator-framework/kubectl-operator/internal/pkg/catalogd/streamer"
+	experimentalaction "github.com/operator-framework/kubectl-operator/internal/pkg/experimental/action"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+	"sigs.k8s.io/yaml"
+
+	"github.com/spf13/cobra"
+)
+
+func NewInspectCommand(cfg *action.Configuration) *cobra.Command {
+	i, err := experimentalaction.NewCatalogInspect(cfg,
+		func(c *action.Configuration) experimentalaction.CatalogFetcher {
+			return fetcher.New(c.Client)
+		},
+		func(c *action.Configuration) experimentalaction.CatalogContentStreamer {
+			return streamer.New(c.Clientset.CoreV1())
+		},
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
If at all possible, it's best to avoid running code that can produce errors as part of the command setup. If an error occurs here, cobra commands are not yet setup, which means:
- Users running an unrelated command will encounter this error
- Users can't run with `--help` and get cobra's usage/help output.

Reply:
Was from https://github.com/operator-framework/kubectl-operator/pull/115#discussion_r1473410860 - should be fixed now